docs/overview.md

```diff
@@ -8,16 +7,15 @@ Mesa is a modular framework for building, analyzing and visualizing agent-based
 
 Mesa is modular, meaning that its modeling, analysis and visualization components are kept separate but intended to work together. The modules are grouped into three categories:
 
-1. **Modeling:** Modules used to build the models themselves: a model and agent classes, a scheduler to determine the sequence in which the agents act, and space for them to move around on.
+1. **Modeling:** Modules used to build the models themselves: a model and agent classes, space for them to move around on, and built-in functionality for managing agents.
```

Comment:
formulation is strange. You shift from modules to classes.

Also, agents move _in_ a space not _on_ a space.

Reply:
Something like this?
```suggestion
1. **Modeling:** Classes used to build the models themselves: a model and agent classes, space for them to move around in, and built-in functionality for managing agents.
```
I didn't want to make too much changes, but I agree it's formulated a bit strangely. If you have specific suggestion please make them! (using the ```suggestion``` syntax).

@jackiekazil @tpike3 maybe one of you have an opinion here from the (original) Mesa conceptual design.